packages/hint-http-cache/src/hint.ts

```diff
@@ -65,34 +65,15 @@ export default class HttpCacheHint implements IHint {
         const maxAgeResource = context.hintOptions && context.hintOptions.maxAgeResource || 31536000; // 1 year
         /** Resources' mediaType that should be cached for a long time.*/
         const longCached = [
-            'application/manifest+json',
-
-            'audio/ogg',
-            'audio/mpeg',
-            'audio/mp4',
+            'audio/*',
```

Comment:
No more code is necessary to make this work?

Reply:
It does seem to fail one jsdom test:

``` text
  › validateResults (/home/vsts/work/1/s/packages/utils-tests-helpers/src/hint-runner.ts:159:11)
  › runHint (/home/vsts/work/1/s/packages/utils-tests-helpers/src/hint-runner.ts:418:20)
```

More below